styles/google-drive/catppuccin.user.less

```
@@ -276,33 +282,18 @@
     .a-b-uoC0bf .ae-uoC0bf .g-d,
     .a-b-uoC0bf
       .ae-mAKE4e-Hc-w-d.h-R-w-d /* File preview (in Drive) share button */ {
-      background-color: @accent;
-      color: @base;
+      background-color: var(--dt-secondary-container);
+      // Should be --dt-on-secondary-container, but doesn't work with filter
+      color: @accent;
       /* File preview (separate tab) share icon */
       .IyROMc-euCgFf-LJSvSb,
       .ae-c /* File preview (in Drive) share icon */ {
-        & when (@flavor = latte) {
-          filter: brightness(0) invert(97%) sepia(4%) saturate(152%) hue-rotate(
-            183deg
-          ) brightness(94%) contrast(106%);
-        }
-        & when (@flavor = frappe) {
-          filter: brightness(0) invert(15%) sepia(8%) saturate(2252%)
-            hue-rotate(192deg) brightness(98%) contrast(85%);
-        }
-        & when (@flavor = macchiato) {
-          filter: brightness(0) invert(13%) sepia(26%) saturate(823%)
-            hue-rotate(195deg) brightness(90%) contrast(92%);
-        }
-        & when (@flavor = mocha) {
-          filter: brightness(0) invert(10%) sepia(36%) saturate(650%)
-            hue-rotate(201deg) brightness(90%) contrast(93%);
-        }
+        filter: @catppuccin-filters[@@flavor][@@accentColor];
```

```suggestion
        filter: @accent-filter;
```

Same with the rest. You aren't meant to be using `@catppuccin-filters`.